docs/core/entity/air-quality.md

```diff
@@ -19,17 +19,16 @@ The Air Quality entity does not support attribute shorthand for [property implem
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| particulate_matter_2_5 | float | **Required** | The particulate matter 2.5 (<= 2.5 μm) level.
-| particulate_matter_10 | float | **Required** | The particulate matter 10 (<= 10 μm) level.
-| particulate_matter_0_1 | float | `None` | The particulate matter 0.1 (<= 0.1 μm) level.
-| air_quality_index | float | `None` | The Air Quality Index (AQI).
-| ozone | float | `None` | The O3 (ozone) level.
-| carbon_monoxide | float | `None` | The CO (carbon monoxide) level.
-| carbon_dioxide | float | `None` | The CO2 (carbon dioxide) level.
-| sulphur_dioxide | float | `None` | The SO2 (sulphur dioxide) level.
-| nitrogen_oxide | float | `None` | The N2O (nitrogen oxide) level.
-| nitrogen_monoxide | float | `None` | The NO (nitrogen monoxide) level.
-| nitrogen_dioxide | float | `None` | The NO2 (nitrogen dioxide) level.
-| volatile_organic_compounds | float | `None` | The volatile organic compounds (VOC) level.
+| particulate_matter_2_5 | <code>str &#124; int &#124; float &#124; None</code> | **Required** | The particulate matter 2.5 (<= 2.5 μm) level.
+| particulate_matter_10 | <code>str &#124; int &#124; float &#124; None</code> | `None` | The particulate matter 10 (<= 10 μm) level.
+| particulate_matter_0_1 | <code>str &#124; int &#124; float &#124; None</code> | `None` | The particulate matter 0.1 (<= 0.1 μm) level.
+| air_quality_index | <code>str &#124; int &#124; float &#124; None</code> | `None` | The Air Quality Index (AQI).
+| ozone | <code>str &#124; int &#124; float &#124; None</code> | `None` | The O3 (ozone) level.
+| carbon_monoxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The CO (carbon monoxide) level.
+| carbon_dioxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The CO2 (carbon dioxide) level.
+| sulphur_dioxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The SO2 (sulphur dioxide) level.
+| nitrogen_oxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The N2O (nitrogen oxide) level.
+| nitrogen_monoxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The NO (nitrogen monoxide) level.
+| nitrogen_dioxide | <code>str &#124; int &#124; float &#124; None</code> | `None` | The NO2 (nitrogen dioxide) level.
```

Comment:
The markup is somewhat unreadable because of the `&#124;` but I can't find another way to make the formatting work.

Reply:
Lucky that netlify deploys are preview :smile: